### DISCUSSION TITLE
Am I Too Newbie to Work With Mermaid? (Syntax Error is Happening)

### DISCUSSION THREAD

**Discussion by furkantaskin** [facebook/docusaurus#8286] [Q&A] [answered]
First, thank you so much for your time and sorry for my knowledge.

After updating to 2.2.0 and following the mermaid instruction located in [diagrams page](https://docusaurus.io/docs/markdown-features/diagrams#configuration) I was still unable to activate Mermaid or use it properly because it gives me Syntax error. An example is [located here](https://furkantaskin.netlify.app/docs/python/print) 

Example mermaid block is like ([located here](https://github.com/furkantaskin/personalwiki/blob/master/docs/python/6-print.mdx))

If you want a detailed information all files are available [here](https://github.com/furkantaskin/personalwiki)

```
    ```mermaid
      graph TB;
        a-->b
    ```
```

Current dependencies in package.json:

```json
"dependencies": {
    "@docusaurus/core": "^2.2.0",
    "@docusaurus/module-type-aliases": "^2.2.0",
    "@docusaurus/preset-classic": "^2.2.0",
    "@docusaurus/theme-mermaid": "^2.2.0",
    "@mdx-js/react": "^1.6.22",
    "clsx": "^1.2.1",
    "prism-react-renderer": "^1.3.5",
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
  },
```

Current configuration in docusaurus.config.js

```js
markdown:{
    mermaid: true,
  },
  themes:['@docusaurus/theme-mermaid'],
```

**Comment by slorber** [comment, marked as answer]
There's a temporary issue due to latest Mermaid release. You can use `mermaid@9.1.7` to make it work (no semver range). See https://github.com/facebook/docusaurus/issues/8274
